src/command.rs

```diff
@@ -36,6 +79,11 @@ where
     C: IntoComponentSource,
 {
     fn write(self: Arc<Self>, world: &mut World) {
+        tracing::trace!(
+            "insert, ({}. {})",
+            std::any::type_name::<T>(),
+            std::any::type_name::<C>()
+        );
         let consumed = Arc::try_unwrap(self).unwrap();
         world.insert(consumed.tags, consumed.components);
```

Comment:
`world.insert` already traces, so users are going to see two log messages for every insertion here (and other command buffer actions which are doing their own tracing). Instead, these type names could be added to the structural data in the existing tracing event if it is needed.